### ISSUE TITLE
Foreground of disabled button in PopupBox not correct

### ISSUE THREAD

**Issue by tstephansen** [ButchersBoy/MaterialDesignInXamlToolkit#423]
When a button is disabled in the PopupBox it has the same foreground color as other buttons. However, the mouse hover doesn't show so the user does have some indication that the button is disabled. Do you have a resource for the disabled button text? If not, I just fixed it with a style trigger. I can submit a PR with the fix if you want me to. 

```
<Style.Triggers>
    <Trigger Property="IsEnabled" Value="False">
         <Setter Property="Foreground" Value="#56000000"/>
    </Trigger>
</Style.Triggers>
```

**Comment by ButchersBoy** [comment]
Hi, this should be fixed in 2.0 -pre already (are you using 1.5?)

**Comment by tstephansen** [comment]
Yes I am. I will try switching to the -pre. I just started a new application recently so it shouldn't be too hard to make the necessary adjustments (if any). I'll go ahead and close this issue. Thank you!
